sklearn/multioutput.py

```diff
@@ -714,6 +721,28 @@ def predict(self, X):
 
         return Y_pred
 
+    def get_metadata_routing(self):
+        """Get metadata routing of this object.
+
+        Please check :ref:`User Guide <metadata_routing>` on how the routing
+        mechanism works.
+
+        Returns
+        -------
+        routing : MetadataRouter
+            A :class:`~utils.metadata_routing.MetadataRouter` encapsulating
+            routing information.
+        """
+        router = (
+            MetadataRouter(owner=self.__class__.__name__)
+            .add(
+                estimator=self.base_estimator,
+                method_mapping=MethodMapping().add(callee="fit", caller="fit"),
+            )
+            .warn_on(child="estimator", method="fit", params=None)
```

Comment:
This method (`get_metadata_routing`) needs to be defined in the child classes because one of them (RegressorChain) already accepts and passes `fit_params` (hence the warning here correct), but the ClassifierChain doesn't and passing `fit_params` is added in this PR and therefore shouldn't add this warning.